package.json

```diff
@@ -39,8 +39,8 @@
     "lodash": "^4.17.21",
     "query-string": "^6.11.1",
     "web3": "0.20.7",
-    "wyvern-js": "git+https://github.com/ProjectOpenSea/wyvern-js.git#v3.3.4",
-    "wyvern-schemas": "git+https://github.com/ProjectOpenSea/wyvern-schemas.git#v0.14.0"
+    "wyvern-js": "git+https://github.com/ProjectOpenSea/wyvern-js.git#adb0b5ceb8616ce1414096a27c0a8a953120c0aa",
```

Comment:
can we use git tags and versions for these? more readable and reviewable than commit hashes

Reply:
I'm waiting for those PRs on the other repos to be approved and merged before doing that :P